flent/plotters.py

```diff
@@ -1238,7 +1238,10 @@ def _do_scaling(self, axis, data, btm, top, unit=None, allow_log=True):
             axis.set_ylim(0, top_scale)
         else:
             if self.log_base:
-                axis.set_yscale('log', basey=self.log_base)
+                if MPL_VER >= LooseVersion("3.3"):
```

Comment:
As of commit 3fb95b089917ca05b58169a055204f82039f35f2 we no longer have the LooseVersion available. I think it's better to have this check use duck typing; something like:

```python
try:
  axis.set_yscale('log', base=self.log_base)
except TypeError:
  axis.set_yscale('log', basey=self.log_base)
```

Reply:
I've made the update and tested it again. It also worked without issues with Python 8 and Matplotlib 3.2.2.